### PR TITLE
Cosmos Spark Connector release 4.1.0

### PR DESCRIPTION
**Pull request by FabianMeiswinkel**

### CODE REVIEW

**Review by moderakh — APPROVED** [review]
LGTM. Thanks Fabian

File: sdk/cosmos/azure-cosmos-spark_3-1_2-12/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 ## Release History
 
-### 4.1.0-beta.1 (Unreleased)
+### 4.1.0-beta.1 (2021-05-27)
```
Comment (moderakh) [inline sdk/cosmos/azure-cosmos-spark_3-1_2-12/CHANGELOG.md:3]: shouldn't this be `4.1.0` ?

Comment (moderakh) [inline sdk/cosmos/azure-cosmos-spark_3-1_2-12/CHANGELOG.md:3]:
```suggestion
### 4.1.0 (2021-05-27)
```

Comment (moderakh) [inline sdk/cosmos/azure-cosmos-spark_3-1_2-12/CHANGELOG.md:3]: also please update Readme.md file once release goes on maven:
https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/azure-cosmos-spark_3-1_2-12/README.md

Comment (FabianMeiswinkel) [reply sdk/cosmos/azure-cosmos-spark_3-1_2-12/CHANGELOG.md]: ACK

Comment (FabianMeiswinkel) [reply sdk/cosmos/azure-cosmos-spark_3-1_2-12/CHANGELOG.md]: ACK

**Review by kushagraThapar — APPROVED** [review]
LGTM

**Pull request merged**